operator/src/main/java/org/bf2/operator/operands/KafkaInstanceConfigurations.java

```diff
@@ -44,7 +51,7 @@ void init() throws IOException {
         loadConfiguration(defaultValues, MANAGEDKAFKA);
 
         for (InstanceType type : InstanceType.values()) {
-            configs.put(type, loadConfiguration(new HashMap<>(defaultValues), type.name().toLowerCase()));
+            configs.put(type.name().toLowerCase(), loadConfiguration(new HashMap<>(defaultValues), type.name().toLowerCase()));
```

Comment:
multiple types of usage of this lower cased name I see, maybe better to pull all these into single method?

Reply:
Sorry, hadn't switched those over to lowerName yet.